### DISCUSSION TITLE
tutorials/voyage-part2/03-monolith-as-a-subgraph

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#73] [course feedback]
# tutorials/voyage-part2/03-monolith-as-a-subgraph

https://www.apollographql.com/tutorials/voyage-part2/03-monolith-as-a-subgraph

**Comment by Vicky310** [comment]
If any one is facing "error[E003]: Invalid header value" while running the rover publish command, you will need to run `rover config auth` again and instead of pressing `Ctrl+V` to paste the API key, you should right-click and press paste
